File: uRides/uRides/PaymentViewController.m
//
//  PaymentViewController.m
//  uRides
//
//  Created by Colin Targonski on 12/25/15.
//  Copyright © 2015 uRides Corporation. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "PaymentViewController.h"

// PaymentViewController.m

@interface PaymentViewController ()<STPPaymentCardTextFieldDelegate>

@property(nonatomic) STPPaymentCardTextField *paymentTextField;

- (void)viewDidLoad
{
    [super viewDidLoad];
    self.paymentTextField = [[STPPaymentCardTextField alloc] initWithFrame:CGRectMake(15, 15, CGRectGetWidth(self.view.frame) - 30, 44)];
    self.paymentTextField.delegate = self;
    [self.view addSubview:self.paymentTextField];
}

- (void)paymentCardTextFieldDidChange:(STPPaymentCardTextField *)textField {
    {
        // Toggle navigation, for example
        self.saveButton.enabled = textField.isValid;
    }
    
@end

File: uRides/uRides/PaymentViewController.swift
//
//  PaymentViewController.swift
//
//  Created by Colin Targonski on 12/29/15.
//

import UIKit
import Stripe

class PaymentViewController: UIViewController, STPPaymentCardTextFieldDelegate {
    
    @IBOutlet weak var paymentTextField: STPPaymentCardTextField!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.paymentTextField = STPPaymentCardTextField(frame: CGRectMake(15, 15, 370, 44))
        
        //self.paymentTextField.delegate = self
        
        //self.view!.addSubview(self.paymentTextField)

    }
    
    
    
}